config.yaml

```diff
@@ -52,24 +52,26 @@ commands:
     rules: [ 'allow' ]
 templates:
   comps_list:
-    slack: |
-      {{name}}
+    body: |
+      ~$results[0].name~
```

Comment:
This seems weird. Are we really only ever listing a single component here? I think the previous template ran for each item in the result where this just runs once. Probably need to wrap in:

```
~each var=$results as=result~
~end~
```